tests/integration/security/authorization_test.go

```diff
@@ -75,49 +75,52 @@ func TestAuthorization_mTLS(t *testing.T) {
 	framework.NewTest(t).
 		Features("security.authorization.mtls-local").
 		Run(func(t framework.TestContext) {
-			args := map[string]string{
-				"Namespace":  apps.Namespace1.Name(),
-				"Namespace2": apps.Namespace2.Name(),
-			}
-			policies := tmpl.EvaluateAllOrFail(t, args,
-				file.AsStringOrFail(t, "testdata/authz/v1beta1-mtls.yaml.tmpl"))
-
-			t.Config().ApplyYAMLOrFail(t, apps.Namespace1.Name(), policies...)
-			for _, cluster := range t.Clusters() {
-				t.NewSubTest(fmt.Sprintf("From %s", cluster.StableName())).Run(func(t framework.TestContext) {
-					a := apps.A.Match(echo.InCluster(cluster).And(echo.Namespace(apps.Namespace1.Name())))
-					c := apps.C.Match(echo.InCluster(cluster).And(echo.Namespace(apps.Namespace2.Name())))
-					newTestCase := func(from echo.Instance, path string, expectAllowed bool) rbacUtil.TestCase {
-						callCount := 1
-						destInstances := apps.B.Match(echo.Namespace(apps.Namespace1.Name()))
-						if destInstances.Clusters().IsMulticluster() {
-							// so we can validate all clusters are hit
-							callCount = util.CallsPerCluster * len(destInstances.Clusters())
-						}
-						return rbacUtil.TestCase{
-							Request: connection.Checker{
-								From: from, // From service
-								Options: echo.CallOptions{
-									Target:   destInstances[0],
-									PortName: "http",
-									Scheme:   scheme.HTTP,
-									Path:     path, // Requested path
-									Count:    callCount,
+			b := apps.B.Match(echo.Namespace(apps.Namespace1.Name()))
+			vm := apps.VM.Match(echo.Namespace(apps.Namespace1.Name()))
+			for _, dst := range []echo.Instances{b, vm} {
+				args := map[string]string{
+					"Namespace":  apps.Namespace1.Name(),
+					"Namespace2": apps.Namespace2.Name(),
+					"dst":        dst[0].Config().Service,
+				}
+				policies := tmpl.EvaluateAllOrFail(t, args,
+					file.AsStringOrFail(t, "testdata/authz/v1beta1-mtls.yaml.tmpl"))
+				t.Config().ApplyYAMLOrFail(t, apps.Namespace1.Name(), policies...)
```

Comment:
are the policies deleted after this test?

Reply:
Yes, with https://github.com/istio/istio/pull/30338 in master branch, policies are auto cleaned up.